packages/@tinacms/mdx/src/next/tests/mdx-inline-nested/field.ts

```diff
@@ -0,0 +1,43 @@
+import { RichTextField } from '@tinacms/schema-tools'
+
+export const field: RichTextField = {
+  name: 'body',
+  type: 'rich-text',
+  parser: { type: 'mdx' },
+  templates: [
+    {
+      name: 'dropBox',
+      label: 'DropBox',
+      inline: true,
+      fields: [
+        {
+          name: 'title',
+          label: 'Short Description',
+          type: 'string',
+          required: true,
+        },
+        {
+          name: 'body',
+          label: 'Expanded Information',
+          type: 'rich-text',
+          required: true,
+          templates: [
+            {
+              name: 'dropBox',
+              label: 'DropBox',
+              inline: true,
+              fields: [
+                {
+                  name: 'title',
+                  label: 'Short Description',
+                  type: 'string',
+                  required: true,
+                },
+              ],
+            },
+          ],
```

Comment:
This looks really good. I understand that this would be a different issue, so just as FYI: This content structure will fail in the editor when clicking on the nested `dropBox`. The first one is fine, the 2nd will throw a lot of JS errors in the browser console and render a white screen.